packages/block-library/src/button/edit.native.js

```diff
@@ -288,13 +319,41 @@ class ButtonEdit extends Component {
 		this.richTextRef = richText;
 	}
 
+	onReplaceSubsheet( destination, callback ) {
+		const { stack } = this.state;
```

Comment:
I think that is should be more reusable. Adding a color palette to the block should be done in an easier way. I don't have enough context at this moment but let's try to find a better solution

Reply:
Decided to move that functionality into BottomSheet, but we can look closer at it, maybe we'll find a better way to handle it :)